pkg/skaffold/sync/sync_test.go

```diff
@@ -799,6 +799,16 @@ func TestIntersect(t *testing.T) {
 				"index.html": {"/html/index.html"},
 			},
 		},
+		{
+			description: "absolute path",
+			files:       []string{"/home/test/example.html"},
```

Comment:
We need some more tests including files that shouldn't match like `/etc/resolv.conf` and `/usr/bin/newfile`.

Reply:
Can you specify this?

Reply:
Meaning that there should be tests with absolute files that don't match a sync rule, to verify that the absolute files aren't sent unnecessarily.  So specifically something like this:
```
		{
			description: "only matching absolute paths are sent",
			files:       []string{"/an/absolute/file"},
 			syncRules: []*latestV1.SyncRule{
				{Src: "/home/test/**", Dest: "/html"},
			},
			expected: nil,
		},
```

Reply:
Got it

Reply:
Added 2 more test cases